src/test/java/com/ebivariation/contigalias/NCBIBrowserTest.java

```diff
@@ -0,0 +1,87 @@
+package com.ebivariation.contigalias;
+
+import com.ebivariation.contigalias.dus.NCBIBrowser;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class NCBIBrowserTest {
+
+    @Test
+    void connect() throws IOException {
+        NCBIBrowser ncbiBrowser = new NCBIBrowser();
+        try {
+            ncbiBrowser.connect();
+        } finally {
+            ncbiBrowser.disconnect();
+        }
+    }
+
+    @Test
+    void navigateToAllGenomesDirectory() throws IOException {
+        NCBIBrowser ncbiBrowser = new NCBIBrowser();
+        try {
+            ncbiBrowser.connect();
+            ncbiBrowser.navigateToAllGenomesDirectory();
+            assertTrue(ncbiBrowser.listFiles().length > 0);
+        } finally {
+            ncbiBrowser.disconnect();
+        }
+    }
+
+    @Test
+    void navigateToSubDirectoryPath() throws IOException {
+        NCBIBrowser ncbiBrowser = new NCBIBrowser();
+        try {
+            ncbiBrowser.connect();
+            ncbiBrowser.navigateToSubDirectoryPath("/genomes/INFLUENZA/");
+            assertTrue(ncbiBrowser.listFiles().length > 0);
+        } finally {
+            ncbiBrowser.disconnect();
+        }
+    }
+
+    @Test
+    void getGenomeReportDirectory_GCATest() throws IOException {
+        NCBIBrowser ncbiBrowser = new NCBIBrowser();
+        try {
+            ncbiBrowser.connect();
+            String path = ncbiBrowser.getGenomeReportDirectory("GCA_004051055.1");
+            assertEquals("GCA/004/051/055/GCA_004051055.1_ASM405105v1/", path);
+        } finally {
+            ncbiBrowser.disconnect();
+        }
+    }
+
+    @Test
+    void getGenomeReportDirectory_GCFTest() throws IOException {
+        NCBIBrowser ncbiBrowser = new NCBIBrowser();
+        try {
+            ncbiBrowser.connect();
+            String path = ncbiBrowser.getGenomeReportDirectory("GCF_007608995.1");
+            assertEquals("GCF/007/608/995/GCF_007608995.1_ASM760899v1/", path);
+        } finally {
+            ncbiBrowser.disconnect();
+        }
+    }
+
+    @Test
+    void getAssemblyReportInputStream() throws IOException {
+        NCBIBrowser ncbiBrowser = new NCBIBrowser();
+        try {
+            ncbiBrowser.connect();
+            InputStream stream = ncbiBrowser.getAssemblyReportInputStream("/genomes/all/GCF/007/608/995/GCF_007608995.1_ASM760899v1/");
+            int available = stream.available();
+            if (available > 0) {
+                stream.close();
+                System.out.println("Stream closed with availability: " + available);
```

Comment:
don't put system.prints in tests. just do assertions that fail if there's anything wrong, and say nothing if it's ok.